dwitter/migrations/0013_auto_20170226_1249.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('dwitter', '0012_auto_20170225_2056'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
```

Comment:
Removes author and creates a new field _author that is just set to null. Posting new dweets work though, and the soft delete seems to work :)